brand/views.py

```diff
@@ -1,4 +1,117 @@
-from django.shortcuts import render
+from unicodedata import name
+from uuid import uuid4
+from dal import autocomplete
+
+from django.shortcuts import render, get_object_or_404, redirect
+from django.http import HttpResponse
+from django.views.generic import CreateView
+from django.forms.models import model_to_dict
+from django.forms import (
+    ModelForm,
+    ModelMultipleChoiceField,
+    inlineformset_factory,
+    ModelChoiceField,
+    Form,
+)
+
+from brand.models import brand
+from .models import Brand, BrandUpdate, BrandFeature
+from .forms import CreateUpdateForm, BrandFeaturesForm
+
+from cities_light.models import Region, SubRegion
+
+
+class RegionAutocomplete(autocomplete.Select2QuerySetView):
+    def get_queryset(self):
+        # Don't forget to filter out results depending on the visitor !
+        if not self.request.user.is_authenticated:
+            return Region.objects.all()
+
+        qs = Region.objects.all()
+
+        if self.q:
+            qs = qs.filter(name__icontains=self.q)
+
+        return qs
+
+
+class SubRegionAutocomplete(autocomplete.Select2QuerySetView):
+    def get_queryset(self):
+        # Don't forget to filter out results depending on the visitor !
+        if not self.request.user.is_authenticated:
+            return SubRegion.objects.all()
+
+        qs = SubRegion.objects.all()
+
+        if self.q:
+            qs = qs.filter(name__icontains=self.q)
+
+        return qs
+
+
+
+class CreateUpdateView(CreateView):
+    template_name = "update.html"
+    form_class = CreateUpdateForm
+    success_url = "https://bank.green"
```

Comment:
Could you please put in an intermediary thank you view with a few seconds to redirect?

"""
💚 Thank you for helping improve Bank.Green's data and making the world a greener place. 💚

Your suggestion is being reviewed and we'll get in contact with you if anything is unclear. Your changes will most likely appear in the next week.

You will now be redirected to the page you were on.
"